pkg/installation.go

```diff
@@ -0,0 +1,65 @@
+package pkg
```

Comment:
or we could move this file to the `pkg/toolchain` pkg, which is another name for `util` or `common`, isn't it?

Reply:
The `pkg/toolchain` pkg was the first place where I put it, but then I got dependency cycle failure :-/

Reply:
damn'it!